tests/test_venue_request_v2.py

```diff
@@ -2131,6 +2258,9 @@ def test_release_meta_reviews_to_authors_and_reviewers(self, test_client, helper
         assert 'V2.cc/2030/Conference/Submission1/Authors' in meta_reviews[0].readers
         assert len(meta_reviews[0].nonreaders) == 0
 
+        assert 'final_recommendation' in meta_reviews[0].content
+        assert 'readers' in meta_reviews[0].content['final_recommendation']
```

Comment:
@melisabok this is what I meant about the issue with re-running the stage from the request form. 

In this case, a PC could re-run the metareview stage to release metareviews to authors. Since `final_recommendation` was added to metareviews in a metareview revision, this field is not in the metareview additional fields in the request form. This means that the readers is removed from this field when re-running the metareview. 
Not sure if this is an issue but something we would have to keep in mind.